### PR TITLE
Sort map keys using a natural sorting algorithm.

### LINKED ISSUE #40: Use natural sorting for map keys.
For example, if the key type is numeric use a numeric sort.

### PR DESCRIPTION
**Pull request by jmalloc**
Fixes #40.

### CODE REVIEW

File: map.go [hunk 1]
```diff
@@ -131,7 +132,7 @@ func (r *mapRenderer) finalize() {
 	sort.Slice(
 		r.pairs,
 		func(i, j int) bool {
-			return r.pairs[i].Key < r.pairs[j].Key
+			return natsort.Less(r.pairs[i].Key, r.pairs[j].Key)
```
Comment (jmalloc) [inline map.go]: We just need some tests to ensure that the map rendering itself is sorting how we intend.

Comment (danilvpetrov) [reply map.go]: cool, can do.

Comment (danilvpetrov) [reply map.go]: All right the first iteration of tests are in.

**Comment by jmalloc** [comment]
LGTM, I can't "approve" since I created the PR, so merge when ready.

**Comment by danilvpetrov** [comment]
Thanks guys!

**Pull request merged**